R/helpers.R

```diff
@@ -56,27 +58,29 @@ use_dependency <- function(package, type, min_version = NULL) {
   delta <- sign(match(existing_type, types) - match(type, types))
   if (delta < 0) {
     # don't downgrade
-    ui_warn(
```

Comment:
`ui_warn()` was seldom-used and we don't love warnings anyway. So I replaced all instances of it with `ui_bullets(c("!" = "I need to warn you about this thing.")`.